portable/CMakeLists.txt

```diff
@@ -0,0 +1,276 @@
+# FreeRTOS internal cmake file. Do not use it in user top-level project
+
+add_library(freertos_kernel_port STATIC
+    $<$<STREQUAL:${FREERTOS_PORT},BCC_16BIT_DOS_FLSH186>:BCC/16BitDOS/common/portcomn.c;BCC/16BitDOS/Flsh186/port.c>
```

Comment:
Can we limit this to one source file per line to make the git history more readable?

Reply:
Yes. Fixed.